app/scripts/app.js

```diff
@@ -153,6 +153,18 @@ angular.module('dashboard', [
             return $rootScope.menu === CONST.MENU.FULL;
         };
 
+        $rootScope.successInfo = function (info) {
```

Comment:
For util functionas generally speaking shouldn't a service be preferred over rootscope?